mapper/mapper_test.go

```diff
@@ -250,6 +264,21 @@ func TestUnitMapper(t *testing.T) {
 			ImageURL:    "",
 		},
 	}
+
+	var mockedAroundONS = []model.Feature{
```

Comment:
Consider adding some additional test cased to over scenarios where this is an empty struct or has some/all empty fields. I think it should be OK but tests will prove that and will safeguard us if the behaviour changes in future.

Reply:
added tests